dartagnan/src/main/java/com/dat3m/dartagnan/parsers/cat/VisitorCat.java

```diff
@@ -62,6 +63,13 @@ public String toString() {
     VisitorCat(Path includePath) {
         this.includePath = includePath;
         this.wmm = new Wmm();
+        try {
+            // The standard library is a cat file stdlib.cat which all models include by default
+            final CatParser parser = getParser(CharStreams.fromPath(Path.of(GlobalSettings.getCatDirectory() + "/stdlib.cat")));
+            parser.mcm().accept(this);
+        } catch (IOException e) {
+            throw new ParsingException("Error parsing stdlib.cat file", e);
+        }
```

Comment:
Can you put this into a separate function like "includeStdlib". I'm not a big fan of having complex logic sitting around in the constructor.